src/app/Http/Controllers/UserController.php

```diff
@@ -2,19 +2,24 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\RedirectResponse;
+use App\Models\Post;
 use Illuminate\View\View;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Auth;
 
 class UserController extends Controller
 {
     public function show(): View
     {
-        return view('user/profile');
+        $user = Auth::user();
```

Comment:
Do you think you are using $user? If not, remove it.

Reply:
I have resolved this comment, sir. Thank you! 🙇